### DISCUSSION TITLE
Welcome to Question & Answer Discussion

### DISCUSSION THREAD

**Discussion by balaji-sivasakthi** [balaji-sivasakthi/regex-extension#53] [Q&A] [answered]
Discussion related to chrome extension

**Comment by balaji-sivasakthi** [comment]
How to debug "Missing font" In Google chrome

**Reply by Gobinath-B** [reply, marked as answer]
> The updated manifest.json:

 ```
 "content_scripts": [
    {
      "matches": ["http://*/*", "https://*/*"], 
      "all_frames": true,
      "css" : ["fonts.css"]
    }
  ],
```
A coworker hinted at the issue, and I confirmed it when I tried to use a font that was added to a parent iframe and not to the one that I wanted.

As for "How to debug" these kinds of issues, that still doesn't have a clear answer. I could not find a way to list which fonts were available for a given frame, nor to see whether the injected CSS was installed on that frame.
